### DISCUSSION TITLE
kf.notifications: env says KDE is running but SNI unavailable -- check KDE_FULL_SESSION and XDG_CURRENT_DESKTOP

### DISCUSSION THREAD

**Discussion by eprparadocs** [evilsocket/opensnitch#792] [Q&A]
I followed the installation and started the opensnitchd running. When I try to start up the GUI I get this message:

kf.notifications: env says KDE is running but SNI unavailable -- check KDE_FULL_SESSION and XDG_CURRENT_DESKTOP

I searched for an answer but can't find anything. When I check KDE_FULLSESSION it shows true, and XDG_CURRENT_DESKTOP shows KDE.  What am I supposed to do to correct whatever the problem is?

**Comment by gustavo-iniguez-goya** [comment]
Hi @eprparadocs ,

As far as I can tell, that error is related to KDE not OpenSnitch. I've found the same error related to keepassxc, pamac-tray-plasma and msms-kde-notifier (due to a broken Manjaro update):
```
11/12/22 5:01 PM | keepassxc | kf.notifications: env says KDE is running but SNI unavailable – check KDE_FULL_SESSION and XDG_CURRENT_DESKTOP
11/12/22 5:01 PM | pamac-tray-plasma | kf.notifications: env says KDE is running but SNI unavailable – check KDE_FULL_SESSION and XDG_CURRENT_DESKTOP
11/12/22 5:01 PM | msm_kde_notifier | kf.notifications: env says KDE is running but SNI unavailable – check KDE_FULL_SESSION and XDG_CURRENT_DESKTOP

```
https://forum.manjaro.org/t/pan-vpn-connection-fails-after-an-update-again/125940/8

Could you provide the following information?:
- Linux Distribution
- kernel
- OpenSnitch version
- Link to the installation instructions
- Does the GUI appear despite the error? (see if you have the OpenSnitch icon in your systray)

Thank you!

**Reply by eprparadocs** [reply]
Linux distro

Distributor ID: Ubuntu
Description:    Ubuntu 22.10
Release:        22.10
Codename:       kinetic


Kernel:  5.19.0-26-generic 

Opensnitch version:  opensnitch_1.6.0-rc.3-1_amd64
UI version:  ython3-opensnitch-ui_1.6.0-rc.3-1_all

installation instructions: https://github.com/evilsocket/opensnitch

GUI doesn't display at all.

Thanks and peace!
Chuck

**Reply by eprparadocs** [reply]
I figured it all out. It turns out the opensnithchd didn't startup automatically. I did it by hand and the message went away!  I also added the two pip3 installs that I needed and now everything works.

Thanks for answering!

**Reply by gustavo-iniguez-goya** [reply]
yes! :tada:  :)
